src/main/java/io/lenses/connect/smt/header/InsertWallclock.java

```diff
@@ -154,6 +165,13 @@ public void configure(Map<String, ?> props) {
               + ConfigName.VALUE_TYPE
               + "' must be set to either 'epoch' or 'format'.");
     }
+    final String timezoneStr = config.getString(ConfigName.TIMEZONE);
+    try {
+      timeZone = TimeZone.getTimeZone(timezoneStr);
+    } catch (IllegalArgumentException e) {
+      throw new ConfigException(
+          "Configuration '" + ConfigName.TIMEZONE + "' is not a valid timezone.");
```

Comment:
how about either passing the value instead of key or maybe even:
```
      throw new ConfigException(
          "Configuration '" + ConfigName.TIMEZONE + "=" + "config.getString(ConfigName.TIMEZONE)" + "' is not a valid timezone.");
```
?

Reply:
Actually, the try/catch block is redundant since getTimezone does not raise an exception.